scripts/docgen/snippets/extractSnippets.ts

```diff
@@ -168,6 +153,6 @@ const parseSnipComment = (snipComment: string): ParsedSnip => {
 }
 
 /** Whether a node will be included in snip output */
```

Comment:
I think I wrote this (poor) comment initially to try and clarify that this function was not responsible for actually excluding the node, but now it's either redundant or inaccurate depending on level of precision so let's just delete it 🤣